db/db_local/locations.go

```diff
@@ -48,9 +48,9 @@ func getDataLocation() string {
 	return loc
 }
 
-// location where the dump file will be stored, so that it can be later restored after connections
+// tar file where the dump file will be stored, so that it can be later restored after connections
 // refresh in a new installation
-func getBackupLocation() string {
+func getBackupFile() string {
```

Comment:
can we make all functions in this module consistent with the function names in the filepaths package, i.e. instead of 
```
getPostgresBinaryExecutablePath
```
it would be
```
postgresBinaryExecutableFilePath
```

and this one would be 
```
databaseBackupFilePath
```

Reply:
also at the top of this file is 
```
const ServiceExecutableRelativeLocation = "/db/12.1.0/postgres/bin/postgres"
```

that is not right is it? Can we not autogenerate that path based on current PG version?

Reply:
done